python/lsst/alert/packet/schema.py

```diff
@@ -278,7 +268,7 @@ def __eq__(self, other):
         return self.definition == other.definition
 
     @classmethod
-    def from_file(cls, filename=None, root_name="lsst.alert"):
+    def from_file(cls, filename=None, root_name="lsst.v3_0.alert"):
```

Comment:
This is a little evil, and hardcodes a `v3.0` expectation. That's okay while there's only one version available, though, I think.